js/stateless.js/src/idls/light_account_compression.ts

```diff
@@ -1,6 +1,6 @@
-export type AccountCompression = {
+export type LightAccountCompression = {
     version: '0.3.1';
-    name: 'account_compression';
+    name: 'light_account_compression';
```

Comment:
@vadorovsky do you know why this still has the lightaccountcompression name? maybe need to regenerate idls via `build.sh` script

Reply:
I had to manually remove this file.